aws/resource_aws_spot_fleet_request.go

```diff
@@ -256,6 +256,30 @@ func resourceAwsSpotFleetRequest() *schema.Resource {
 							Computed: true,
 							ForceNew: true,
 						},
+						"tag": {
```

Comment:
I previewed `aws_instance` but with different specs, opted to go with the model used here: https://github.com/terraform-providers/terraform-provider-aws/blob/master/aws/resource_aws_autoscaling_group.go